Applications/Scale/test/testScale.cpp

```diff
@@ -184,3 +185,45 @@ void scaleGait2354_GUI(bool useMarkerPlacement)
 
     delete subject;
 }
+
+void scaleModelWithLigament()
+{
+    // SET OUTPUT FORMATTING
+    IO::SetDigitsPad(4);
+
+    std::string setupFilePath("");
+    ScaleTool* subject;
+    Model* model;
+
+    // Remove old model if any
+    FILE* file2Remove = IO::OpenFile(setupFilePath + "toyLigamentModelScaled.osim", "w");
+    fclose(file2Remove);
+
+    // Construct model and read parameters file
+    subject = new ScaleTool("toyLigamentModel_Setup_Scale.xml");
+
+    // Keep track of the folder containing setup file, wil be used to locate results to comapre against
+    setupFilePath = subject->getPathToSubject();
+
+    model = subject->createModel();
+
+    SimTK::State& s = model->updWorkingState();
+    model->getMultibodySystem().realize(s, SimTK::Stage::Position);
+
+    if (!model) {
+        //throw Exception("scale: ERROR- No model specified.",__FILE__,__LINE__);
+        cout << "scale: ERROR- No model specified.";
+    }
+
+    ASSERT(!subject->isDefaultModelScaler() && subject->getModelScaler().getApply());
+    ModelScaler& scaler = subject->getModelScaler();
+    ASSERT(scaler.processModel(model, setupFilePath, subject->getSubjectMass()));
+
+    const std::string& scaledModelFile = scaler.getOutputModelFileName();
+    const std::string& std_scaledModelFile = "std_toyLigamentModelScaled.osim";
+
+    ASSERT(Model(scaledModelFile) == Model(std_scaledModelFile));
+
```

Comment:
This is setup as a regression test which assumes that ligament scaling was occurring before this fix, which is not the case. If std includes an answer for ligament scaling that came from a path actuator (e.g. muscle) then it is OK, but if the standard was just created then what is being tested?